app/tests/e2e/issuer/test_issuer_ld_bbs.py

```diff
@@ -0,0 +1,382 @@
+        await faber_client.post(
+            CREDENTIALS_BASE_PATH,
+            json=credential,
+        )
+    assert_that(exc.value.status_code).is_equal_to(500)
+
+
+@pytest.mark.anyio
+@pytest.mark.parametrize("register_key_issuer", ["bls12381g2"], indirect=True)
+async def test_send_jsonld_bbs_oob(
+    faber_client: RichAsyncClient,
+    faber_and_alice_connection: FaberAliceConnect,
+    alice_member_client: RichAsyncClient,
+    register_key_issuer: DidKey,
+):
+    alice_connection_id = faber_and_alice_connection.alice_connection_id
+    faber_connection_id = faber_and_alice_connection.faber_connection_id
+
+    response = await alice_member_client.get(
+        CREDENTIALS_BASE_PATH,
+        params={"connection_id": alice_connection_id},
+    )
+    records = response.json()
+
+    # nothing currently in alice's records
+    assert len(records) == 0
+    did = register_key_issuer
+
+    # Updating JSON-LD credential did:sov
+    credential["connection_id"] = faber_connection_id
+    credential["ld_credential_detail"]["credential"]["issuer"] = f"{did}"
+    credential["ld_credential_detail"]["options"] = {"proofType": "BbsBlsSignature2020"}
+
+    # faber create offer
+    response = await faber_client.post(
+        CREDENTIALS_BASE_PATH + "/create-offer",
+        json=credential,
+    )
+
+    data = response.json()
+    assert_that(data).contains("credential_id")
+    assert_that(data).has_state("offer-sent")
+    assert_that(data).has_protocol_version("v2")
+
+    invitation_response = await faber_client.post(
+        OOB_BASE_PATH + "/create-invitation",
+        json={
+            "create_connection": False,
+            "use_public_did": False,
+            "attachments": [
+                {"id": data["credential_id"][3:], "type": "credential-offer"}
+            ],
+        },
+    )
+    assert_that(invitation_response.status_code).is_equal_to(200)
+
+    invitation = (invitation_response.json())["invitation"]
+
+    accept_response = await alice_member_client.post(
+        OOB_BASE_PATH + "/accept-invitation",
+        json={"invitation": invitation},
+    )
+
+    oob_record = accept_response.json()
+
+    assert_that(accept_response.status_code).is_equal_to(200)
+    assert_that(oob_record).contains("created_at", "oob_id", "invitation")
+    assert await check_webhook_state(
+        client=alice_member_client,
+        topic="credentials",
+        filter_map={
+            "state": "offer-received",
+        },
+    )
+
+
+@pytest.mark.anyio
+@pytest.mark.parametrize("register_key_issuer", ["bls12381g2"], indirect=True)
+async def test_send_jsonld_request(
+    alice_member_client: RichAsyncClient,
+    faber_client: RichAsyncClient,
+    faber_and_alice_connection: FaberAliceConnect,
+    register_key_issuer: DidKey,
+):
+    alice_connection_id = faber_and_alice_connection.alice_connection_id
+    faber_connection_id = faber_and_alice_connection.faber_connection_id
+
+    did = register_key_issuer
```

Comment:
In cases where `did` is only used once, to assign `credential["ld_credential_detail"]["credential"]["issuer"] = f"{did}"`, we can just replace that with `credential["ld_credential_detail"]["credential"]["issuer"] = register_key_issuer`